pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -742,6 +742,8 @@ def set_distributed_mode(self, distributed_backend: Optional[str] = None):
 
         # special case with DDP on CPUs
         if self.distributed_backend == "ddp_cpu":
+            if _TPU_AVAILABLE:
+                raise MisconfigurationException("`accelerator='ddp_cpu'` is not supported on TPUs")
```

Comment:
should we say here on TPU machines / TPU platform?

Reply:
well on past TPU you can run ddp_pu as The TPU is attached to a "normal" machine, right?

Reply:
Updated!